app/src/main/java/com/duycuong/weather/ui/screen/main/MainActivity.java

```diff
@@ -0,0 +1,147 @@
+package com.duycuong.weather.ui.screen.main;
+
+import android.content.Intent;
+import android.databinding.BindingAdapter;
```

Comment:
ERROR: Unused import: android.databinding.BindingAdapter. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports